vault/raft_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package vault
+
+import (
+	"testing"
+)
+
+func TestFormatDiscoveredAddr(t *testing.T) {
+	type TestCase struct {
+		addr string
+		port uint
+		res  string
+	}
+	cases := []TestCase{
```

Comment:
This is mostly a question out of curiosity as opposed to something you should change, but out of interest: which of these would have failed with the old implementation?